lib/tools/system-calls.js

```diff
@@ -861,6 +873,17 @@ systemCallMethods.unroot = async function () {
   }
 };
 
+/**
+ * Checks whether the current user is root
+ *
+ * @return {boolean} True if the user is root
+ * @throws {Error} if there was an error while identifying
+ * the user.
+ */
+systemCallMethods.isRoot = async function () {
+  return (await this.shell(['whoami'])).trim() === 'root';
```

Comment:
Could `await this.shell(['whoami'])` ever return 'root' in upper case?

Reply:
I don't think so